### DISCUSSION TITLE
Config Audit Reports Grafana Dashboard

### DISCUSSION THREAD

**Discussion by humpalu** [aquasecurity/trivy-operator#1303] [Development]
Anyone tried already to make a Grafana dashboard for Config Audit Reports like the Detailed Vulnerability dashboards?
Like that: 
https://grafana.com/grafana/dashboards/16337-trivy-operator-vulnerabilities/

I would like to make a dashboard what shows the Config Audit Issues per namespaces and per resources.

**Comment by chen-keinan** [comment]
@tothger its in progress #1302 :)

**Reply by humpalu** [reply]
@chen-keinan Great! Thank you. I will stay tuned :D
